src/app/visual/image/image-visual.component.ts

```diff
@@ -0,0 +1,7 @@
+import { Component } from '@angular/core';
```

Comment:
Import statements need to follow code style guidelines.
See: https://docs.blackbaud.com/skyux-team-docs/style#imports